src/Redis.cpp

```diff
@@ -611,7 +670,11 @@ RCPP_MODULE(Redis) {
         .method("setVector",  &Redis::setVector,   "runs 'SET key object' for a numeric vector")
         .method("getVector",  &Redis::getVector,   "runs 'GET key object' for a numeric vector")
         .method("listLPop",   &Redis::listLPop,    "pops numeric vector to list")
+        .method("LPop",        &Redis::LPop,       "pops R object to list")
+        .method("RPop",        &Redis::RPop,       "pops R object to list")
```

Comment:
I believe I have moved them where you indicated.  Now that I see them in context, I noticed that my description field is inconsistent with the others in this section.  I believe my version reads cleanly enough, but would you prefer that I standardize the language to fit your other methods (or vice-versa)?